src/schematools/types.py

```diff
@@ -1077,3 +1228,16 @@ class Temporal:
 
     identifier: str
     dimensions: Dict[str, Tuple[str, str]] = field(default_factory=dict)
+
+
+def _normalize_scopes(auth: Union[None, str, list, tuple]) -> Set[str]:
+    """Make sure the auth field has a consistent type"""
+    if not auth:
+        # Auth defined on schema
+        return set()
+    elif isinstance(auth, (list, tuple, set)):
```

Comment:
```suggestion
    elif isinstance(auth, Iterable):
```

Reply:
`str` is ook iterable, en dat betekend een set van `{'A', 'U', T', 'H'}`

Reply:
Oh, goed punt.